app/README.md

```diff
@@ -0,0 +1,105 @@
+# App modules
+
+App modules are subsections of the main site, dealing with a specific area , e.g locations or moves. They are constructed using Express routers.
+
+They can have a variety of different parts, in either a root file, or a folder with multiple files, and exposed via an `index.js` in the folder.
+
+- controllers
+  - Express
```

Comment:
Could these maybe use an emdash after the _thing_ instead of a nested list? 

Might be easier to scan the _things_ as a list of bullets with a description after. Semantically they also feel like the same item rather than a child item.

We did something similar in the [framework docs](https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks/tree/master/frameworks/person-escort-record#step-options)